Menu/Tab/index.jsx

```diff
@@ -82,6 +85,16 @@ export default React.createClass({
     }
   },
 
+  componentWillUnmount () {
+    window.removeEventListener('resize', debounce(this.resizeListener))
```

Comment:
Actually, now that I look at the diff, you're not using the debounced listener anywhere except here :P